administrator/components/com_workflow/src/Table/TransitionTable.php

```diff
@@ -58,6 +58,31 @@ public function __construct(DatabaseDriver $db, ?DispatcherInterface $dispatcher
         parent::__construct('#__workflow_transitions', 'id', $db, $dispatcher);
     }
 
+    /**
+     * Overloaded check function
+     *
+     * @return  boolean  True on success
+     *
+     * @see     Table::check()
+     * @since   __DEPLOY_VERSION__
+     */
+    public function check()
+    {
+        try {
+            parent::check();
+        } catch (\Exception $e) {
+            $this->setError($e->getMessage());
+
+            return false;
+        }
```

Comment:
```suggestion
        parent::check();
```
PHPstan error "Call to deprecated method setError()" would be fixed by not doing a try catch here but simply calling the method and let it raise an exception.

I'm not sure though if that would be right.